tariff/octopus/graphql/types.go

```diff
@@ -38,9 +38,22 @@ func (d *tariffData) TariffCode() string {
 	return d.standardTariff.TariffCode
 }
 
-// IsExport is a shortcut function for determining whether the given tariff is for export, regardless of tariff type.
-func (d *tariffData) IsExport() bool {
-	return d.standardTariff.IsExport
+// TariffDirection defines which direction of energy flow is being denoted by the given tariff.
+type TariffDirection string
```

Comment:
that's a little inconsistent. make it either tarifftype or direction, but don't use both